plenty_api/api.py

```diff
@@ -331,6 +332,11 @@ def __plenty_api_request(self,
             time.sleep(3)
 
         logging.debug(f"request url: {raw_response.request.url}")
+        
+        # if we get a file as response, just return the content of raw_respone, so it can be written to a file
```

Comment:
s/raw_respone/raw_response/

And the comment is 110 characters long, which is too long, please restrict the line length to max. 100.